updated clickhouse chart in signoz

medicharlachiranjeevi opened the pull request:
- updated  clickhouse tgz in signoz chart
- Added node selector field in clickhouse values of signoz

prashant-shahi commented:
@medicharlachiranjeevi we have released `clickhouse-16.0.4` chart.
also, we have updated `ClickhouseOperator` to `clickhouseOperator` along with other changes.

I don't think we need the changes introduced in this PR.

medicharlachiranjeevi commented:
- changed ClickhouseOperator to clickhouseOperator 
- and fixed node selector in clickhouse instance as it was in the block of containers it should be outside the containers

prashant-shahi commented:
@medicharlachiranjeevi It seems like there are redundant `nodeSelector`.

```yaml
  {{- if .Values.useNodeSelector }}
  nodeSelector:
    clickhouse: true
  {{- end }}
  {{- if .Values.tolerations }}
  tolerations: {{ toYaml .Values.tolerations | nindent 12 }}
  {{- end }}
  {{- if .Values.nodeSelector }}
  nodeSelector: {{ toYaml .Values.nodeSelector | nindent 12 }}
  {{- end }}
```

The second one seems more generic and gives more control to the users to use existing node labels.
Do let me know which one you think makes more sense to you.

medicharlachiranjeevi commented:
- I agree with u updated

prashant-shahi commented:
@medicharlachiranjeevi Thanks for your PR 🎉
